### ISSUE TITLE
[AnalyticalTable]: Analytical Table In high-contrast themes, texts smaller than 18pt (or 14pt bold) not displayed with a contrast ratio of at least 7:1

### ISSUE THREAD

**Issue by samanr** [UI5/webcomponents-react#7778]
### Describe the bug

When using the SAP High Contrast Theme - Dark, a Success-colored ObjectStatus inside a selected AnalyticalTable row does not meet the required 7:1 foreground/background contrast ratio. The row selection highlight is gold, and the ObjectStatus foreground remains green, resulting in insufficient contrast in the selected state.

### Isolated Example

https://stackblitz.com/edit/github-6bkhssdn?file=src%2FApp.tsx

### Reproduction steps

- Switch the application theme to High Contrast Dark (e.g., sap_horizon_hcb).
- Render an AnalyticalTable with a column that uses ObjectStatus and state="Success" in its cell.
- Select a table row (single- or multi-select).
- Observe the color contrast of the ObjectStatus foreground (green) against the selected row background (gold) -- it doesn’t meet the minimum requirements.

### Expected Behaviour

Should respect the 7:1 contrast ratio.

### Screenshots or Videos

<img width="621" height="285" alt="Image" src="https://github.com/user-attachments/assets/201c433c-6f95-4e26-9e7b-91e272682dab" />

### UI5 Web Components for React Version

2.14.1

### UI5 Web Components Version

2.14.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @samanr 

I couldn't find any specifications regarding this behavior, but I checked the [sap.ui.table.Table](https://sapui5.hana.ondemand.com/#/entity/sap.ui.table.Table) and there the contrast text color is applied in such cases. The linked PR will adjust this in our table as well.

The font-size doesn't seem to change though, neither for the `Text`, `ObjectStatus`, `Tag` etc. so for this to be changed, the global CSS size variable (`--sapFontSize`) would need to be adjusted. As new designs are required, this request would need to go through the Inner Source process and be synchronised with the SAP Central Design team first. Please raise a BLI to the central design team, then open another issue with a small description in our repo again and post the ticket number there.
